web/src/components/shared/SubNavBar/SubNavBar.jsx

```diff
@@ -11,8 +11,8 @@ export default function SubNavBar({
   activeTab,
   app,
   isVeleroInstalled,
-  isAccess,
-  isSnapshots,
+  isAccess = false,
+  isSnapshots = false,
```

Comment:
Give these a default value since they're optional. TS complains if you attempt to use a component without all of it's required props. Props without default values are required.